docs/development.md

```diff
@@ -1,32 +1,136 @@
-$ docker build . -t scpwiki/wikijump:local
-$ docker create --name wikijump -p 80:80 scpwiki/wikijump:local
-$ docker start wikijump
+$ cd installer
+$ docker build . -t scpwiki/wikijump:local --no-cache
 ```
 
-Instead of building Wikijump locally, you can also pull the image from the Docker Hub:
+Then load that image into a container, which is a VM that runs only the image you created:
 
 ```bash
-$ docker create --name wikijump -p 80:80 scpwiki/wikijump:latest
+$ docker create --name wj --publish 80:80 --publish 443:443 -it -d scpwiki/wikijump:local
 ```
 
-Then terminate it:
+The container is already running, but to start the services inside it that run Wikijump you will need to send a start command:
 
 ```bash
-$ docker stop wikijump
+$ docker start wj
 ```
 
-Alternatively, you can install to your local system using `install.sh`. This may require tinkering depending on your exact platform and environment.
+Then navigate to https://www.wikijump.test in your browser.
 
+-----
+
+To stop Wikijump:
+
+```bash
+$ docker stop wj
+```
+
+It's useful to keep track of existing Docker images and containers, and destroy them when you no longer need them, so you don't waste space rebuilding the same image over and over. If you are using Docker Desktop, you can manage containers and images from the GUI. Otherwise, on command line:
+
+```bash
+$ docker container ls  # List containers
+$ docker rm [ID]       # Destroy the container with this ID
+$ docker images        # List images
+$ docker rmi [ID]      # Remove the image with this ID
+```
+
+### Installation via Docker from the Docker Hub
+
+Instead of building Wikijump locally, you can also pull the `latest` image from the Docker Hub:
+
+```bash
+$ docker create --name wj --publish 80:80 --publish 443:443 -it -d scpwiki/wikijump:latest
+```
+
+### Installation via script
+
+You can install to your local system using `install.sh`. This pollutes your system with dependencies and will be difficult to undo. This method is not recommended. It may require tinkering depending on your exact platform and environment.
+
+## Configuration
+
+If you installed Wikijump directly to your machine with `install.sh`, you can edit the Wikijump config at any point. If you installed Wikijump via Docker, you will need to enter the container to edit the config:
+
+```bash
+$ docker exec -it wj bash
+```
 
 You will likely want to set `allow_http` in `/var/www/conf/wikijump.ini` to `true`, since HTTPS locally is troublesome.
 
 If you use a port other than 80 for your Docker container, you will need to edit `site.custom_domain` to add the port number (e.g. "`www.wikijump.test:8080`"). Alternatively, use curl to set the domain directly (e.g. "`-H 'www.wikijump.test'`")
+
+After editing the Wikijump config, you may need to restart nginx.
+
+On Linux with systemd:
+
+```bash
+# systemctl restart nginx
+```
+
+On Linux with sysv-init, Windows via WSL2, or from within a Docker container:
+
+```bash
+# service nginx restart
```

Comment:
These get formatted as comments rather than root shells, would change to just \`\`\`